tests_pytest/common_tests/unit_tests/target_platform_capabilities/test_tpc.py

```diff
@@ -88,6 +108,17 @@ def test_valid_model_object(self, tpc):
         result = load_target_platform_capabilities(tpc)
         assert result == tpc
 
+    def test_new_schema(self, tpc):
+        """Tests that current schema is in all schemas list. This test validates new schema was added properly."""
+        assert type(tpc) in all_tpc_types, "Current schema need to be added to ALL_SCHEMA_VERSIONS"
```

Comment:
This is the relevant code under test:
```
ALL_SCHEMA_VERSIONS = [schema_v1]  # needs to be updated with all active schema versions
all_tpc_types = tuple([s.TargetPlatformCapabilities for s in ALL_SCHEMA_VERSIONS])
```
I’m not sure this achieves the intended behavior. It seems like something might be missing, but perhaps I’m misunderstanding the context. Let’s discuss